prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -5233,6 +5269,14 @@ subroutine getGradients(env, sccCalc, tExtField, isXlbomd, nonSccDeriv, EField,
       end if
     end if
 
+    if (allocated(solvation)) then
+      if (isXlbomd) then
+        call error("XLBOMD does not work with solvation yet!")
```

Comment:
Needs equivalent tests for use with transport or excited states (but in the longer term would be good to get these working with the Born model).